deploy_widget.sh

```diff
@@ -4,12 +4,12 @@ then
   deploy_dir=dist_widget
 fi
 echo "Deploying to $deploy_dir"
-if [ ! -e $deploy_dir ]
-then
-  mkdir $deploy_dir
-  mkdir $deploy_dir/dist
-fi
+
+mkdir -p $deploy_dir/dist
+mkdir -p $deploy_dir/../vendor/d3-3.4.13/ # TODO this doesn't seem right but that's the value in app/js/arethusa.dep_tree/services/dep_tree.js
```

Comment:
In `app/js/arethusa.dep_tree/services/dep_tree.js` the path to `d3` is `../../vendor/d3-3.4.13/d3.js`. Does that mean that when it is deployed it looks for d3 at that location?

Reply:
I struggled with getting this path right.  I was trying to match what you had in the treebank-template with what Arethusa expected. The only way I could get it work was to use the relative path in the arethusa dep_tree.js file.

Reply:
Do you think it makes more sense to be in `$deploy_dir/../vendor/` or `$deploy_dir/vendor`? We can change the Treebank Template to use whatever path makes the most sense here.

Reply:
I have no idea :-)

Reply:
I'll keep it as is then!